klippy/extras/tmc5160.py

```diff
@@ -257,24 +257,26 @@
 ######################################################################
 
 VREF = 0.325
-MAX_CURRENT = 3.000
 
 class TMC5160CurrentHelper:
     def __init__(self, config, mcu_tmc):
         self.printer = config.get_printer()
         self.name = config.get_name().split()[-1]
         self.mcu_tmc = mcu_tmc
         self.fields = mcu_tmc.get_fields()
+        self.sense_resistor = config.getfloat('sense_resistor', 0.075, above=0.)
+        max_current = self._calc_max_current()
         run_current = config.getfloat('run_current',
-                                      above=0., maxval=MAX_CURRENT)
-        hold_current = config.getfloat('hold_current', MAX_CURRENT,
-                                       above=0., maxval=MAX_CURRENT)
+                                      above=0., maxval=max_current)
+        hold_current = config.getfloat('hold_current', max_current,
+                                       above=0., maxval=max_current)
```

Comment:
@bigtreetech 

May I ask out of curiosity: 
What is the purpose of changing `MAX_CURRENT` to `max_current`? 

Is it merely because MAX_CURRENT was a defined directive, compared to the lowercase one being a defined variable in the config, under `__INIT__`? As seen on line 268